jdaviz/core/loaders/importers/spectrum_list/spectrum_list.py

```diff
@@ -23,6 +40,89 @@ def __init__(self, *args, **kwargs):
 
+        self.previous_data_label_messages = []
+
+        if self.is_valid:
+            # If the resolver format is set to "1D Spectrum List", then we
+            # only enable the import button if at least one spectrum is selected.
+            # This is to avoid a user accidentally importing all spectra (API default)
+            # when none are selected
+            self.resolver.observe(self._on_format_selected_change, names='format_selected')
+            # Separately observe changes to the selected spectra
+            self.observe(self._on_spectra_selected_change, names='spectra_selected')
+
+            exposures = []
+            spectra_options = []
+
+            if isinstance(self.input, Spectrum):
+                speclist_input = SpectrumList(self.input_to_list_of_spec(self.input))
+            else:
+                speclist_input = self.input
+
+            for index, spec in enumerate(speclist_input):
+                if self.is_wfssmulti(spec):
+                    # ver, name are stand-ins for exposure and source_id
+                    # ver == exposure, name == source_id
+                    ver, name = self._extract_exposure_sourceid(spec)
+                    exposure_label = f"Exposure {ver}"
+                    exposures.append(exposure_label)
+
+                    label = f"{exposure_label}, Source ID: {name}"
+                    # Flipping the two from the variable naming convention
+                    name_ver = f"{ver}_{name}"
+                    suffix = f"EXP-{ver}_ID-{name}"
+
+                else:
+                    name_ver = index
+                    name = index
+                    ver = index
+                    label = f"1D Spectrum at file index: {index}"
+                    suffix = f"file_index-{index}"
+
+                # Use modified index here to access the spectrum properly
+                # per ``selected_obj_dict`` in SelectFileExtensionComponent.
+                # Attempt to indicate to the user (via label and suffix) that
+                # the index is not the same as the file index.
+                spectra_options.append({'label': label,
+                                        'index': index,
+                                        'name': str(name),
+                                        'ver': str(ver),
+                                        'name_ver': str(name_ver),
+                                        'suffix': suffix,
+                                        'obj': self._apply_spectral_mask(spec)})
+
+            self.spectra = SelectFileExtensionComponent(self,
+                                                        items='spectra_items',
+                                                        selected='spectra_selected',
+                                                        multiselect='spectra_multiselect',
+                                                        manual_options=spectra_options)
+
+            self.spectra.selected = []
```

Comment:
I think this should be the default anyways, if not, we can allow passing `default_mode='empty'` through `SelectFileExtension` above